eng/Publishing.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+
+  <ItemGroup>
+    <ItemsToPushToBlobFeed Include="$(ArtifactsDir)nupkgs/*.nupkg"
+                           IsShipping="true"
+                           UploadPathSegment="nuget-client" />
```

Comment:
Why didn't this need a RelativePath as in the other PRs?

Reply:
RelativePath is needed for assets only, not for nuget packages.